### DISCUSSION TITLE
How can I update Faro to 9.3?

### DISCUSSION THREAD

**Discussion by jarac794** [websu-io/websu#23] [General]
a query how the lighthouse 9.3 could be updated, would you have a guide I would greatly appreciate it :)

**Comment by samos123** [comment]
I don't have that nor planning to in the short-term. Last time I tried upgrading lighthouse it resulted in lots of required changes to the code. So I'm postponing it for now :( why do you need newer version of lighthouse?

**Comment by jarac794** [comment]
Hello, it's fine without problems, if the new version changes the assessment of each audit https://web.dev/performance-scoring/

more than anything for that
